### DISCUSSION TITLE
Integrate helixbridge.app with li.fi

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#96] [Integration]
@xiaoch05 from https://github.com/helix-bridge can assist, cc @1022ranji

**Comment by hackfisher** [comment]
@nblogist Any idea on this, do you need support or documents from @xiaoch05?

**Comment by nblogist** [comment]
Hi @hackfisher ! Happy to put in the integration request and will ping, depending on the li.fi team's response!

**Comment by nblogist** [comment]
Hi @hackfisher just talked with Sarthak from Li.Fi and he says that they aren't looking to add new bridges till Q2, and He'll reach out when the team wants to integrate Helix Bridge :)
